cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/ApiController.java

```diff
@@ -13,18 +13,448 @@
  */
 package io.mesosphere.mesos.frameworks.cassandra;
 
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.google.common.base.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.StringWriter;
 
 @Path("/")
 public final class ApiController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ApiController.class);
+
+    private final CassandraCluster cluster;
+
+    public ApiController(CassandraCluster cluster) {
+        this.cluster = cluster;
+    }
+
+    @GET
+    @Path("/")
+    @Produces("text/html")
+    public String indexPage() {
+        return "<a href=\"status\">All nodes</a> <br/>" +
```

Comment:
Please return a json object here rather than malformed html. It could just be a map.